src/plugins/accentColor.js

```diff
@@ -0,0 +1,19 @@
+import flattenColorPalette from '../util/flattenColorPalette'
+import toColorValue from '../util/toColorValue'
+
+export default function () {
+  return function ({ matchUtilities, theme, variants }) {
+    matchUtilities(
+      {
+        caret: (value) => {
```

Comment:
This right here is why the tests are failing, you are telling Tailwind to match classes starting with the word "caret" when it should be "accent" 👍🏻

Reply:
🤦  That'd certainly do it. Fixed now.